x-pack/plugins/security_solution/server/endpoint/utils/dump.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { inspect } from 'util';
+
+/**
+ * Safely traverse some content (object, array, etc) and stringify it
+ * @param content
+ * @param depth
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const dump = (content: any, depth = 8): string => {
```

Comment:
Maybe rename this to `stringifyDataStructure` or `stringifyDs`. `dump` seems too generic IMO.

Reply:
🤔 
let me think more about it and maybe address in next PR. I like `stringify()`. We also have a `dump()` in our scripts and I think I noticed Konrad use a variation of that recently as well. Will look to unify the instances and rename it then.